src/main/kotlin/no/nav/syfo/dialogmelding/DialogmeldingService.kt

```diff
@@ -15,13 +15,20 @@ class DialogmeldingService(private val behandlerVarselService: BehandlerVarselSe
             val dialogmeldingSvar = dialogmeldingDTO.toDialogmeldingSvar()
             dialogmeldingSvar.innkallingDialogmoteSvar?.let { innkallingSvar ->
                 log.info("Received innkalling dialogmote svar with msgId: ${dialogmeldingDTO.msgId}")
-                behandlerVarselService.opprettVarselSvar(
+                val varselSvarCreated = behandlerVarselService.opprettVarselSvar(
+                    arbeidstakerPersonIdent = dialogmeldingSvar.arbeidstakerPersonIdent,
                     varseltype = innkallingSvar.foresporselType.getVarselType(),
                     svarType = innkallingSvar.svarType.getDialogmoteSvarType(),
                     svarTekst = innkallingSvar.svarTekst,
                     conversationRef = dialogmeldingSvar.conversationRef,
                     parentRef = dialogmeldingSvar.parentRef,
                 )
+                // TODO: Metrics ?
+                if (varselSvarCreated) {
+                    log.info("Created varsel-svar for innkalling dialogmote svar with msgId: ${dialogmeldingDTO.msgId}")
+                } else {
+                    log.error("Failed to create varsel-svar for innkalling dialogmote svar with msgId: ${dialogmeldingDTO.msgId}")
```

Comment:
Er det over-kill å logge med error her? Kan det ikke tenkes at vi får inn dialogmeldinger i denne sløyfa som ikke er relevante for våre dialogmøter?

Reply:
Jo, men man kommer bare hit dersom dialogmelding har innkallingDialogmoteSvar (dvs med svar- og forespørselkoder som er relevant for våre møter) - se mapping og sjekk på linje 15 og 16, så mener det skal være greit med error da. Det betyr i praksis at vi får inn et svar på innkalling, men ikke finner noe varsel å lage svar for.